packages/graphql/src/schema/validation/validate-document.test.ts

```diff
@@ -180,7 +180,7 @@ describe("validateDocument", () => {
             type Post {
                 id: ID! @id @unique
                 title: String!
-                datetime: DateTime @readonly @timestamp(operations: [CREATE])
+                datetime: DateTime @timestamp(operations: [CREATE])
```

Comment:
Should this not make use of @settable so it uses the same schema?

```suggestion
                datetime: DateTime @timestamp(operations: [CREATE]) @settable(onCreate: false, onUpdate: false)
```

Reply:
Fields with `@timestamp` are read-only by their very nature, so I'm not really sure why this was there in the first place! 😆

Reply:
Ah okay, makes sense!